### DISCUSSION TITLE
Is there a way to blacklist `evince-thumbnailer`?

### DISCUSSION THREAD

**Discussion by txtsd** [lxqt/lxqt#2610] [Q&A] [answered]
I have `evince` installed on my system, but I don't want `pcmanfm-qt` to use `evince-thumbnailer` to generate thumbnails.

Is there a way I can blacklist it?

**Comment by tsujan** [comment, marked as answer]
Copy any thumbnailer that you want to disable from `/usr/share/thumbnailers` to `~/.local/share/thumbnailers`, change its `MimeType` line to `MimeType=application/nothing;`, and restart pcmanfm-qt from LXQt Session Settings.

**Reply by tsujan** [reply]
Added the answer to Wiki: https://github.com/lxqt/pcmanfm-qt/wiki#thumbnails

**Reply by txtsd** [reply]
Thanks!
